Add documentation about the Kubernetes Github Flow.

quinton-hoole opened the pull request:
Added an animation (and a link to it) detailing the standard Kubernetes Github Flow.

Commit:
Add documentation about the Kubernetes Github Flow.

Added an animation (and a link to it) detailing the standard Kubernetes Github Flow.

thockin commented on docs/devel/development.md:
```diff
@@ -22,7 +22,9 @@ $ git clone https://github.com/GoogleCloudPlatform/kubernetes.git
 
-(Obviously, clone your own fork of Kubernetes if you plan to do development.)
+If you plan to do development, read about the
+[Kubernetes Github Flow](https://docs.google.com/a/google.com/presentation/d/1WDGN_ggq1Ae3eeQmbSCMyUG1UhhRH6UZTy0pePq09Xo/pub?start=false&loop=false&delayms=3000),
```
This presentation is not visible outside google.com - it's a restriction of our domain.  You can either create it from a private gmail account or render to a less beautiful format like multi-page PDF or maybe animated gif?

thockin replied:
is your slides you said "push request" but meant "pull request"

quinton-hoole replied:
Both fixed.

Commit:
Make slides visible to the public, fix a typo.

Moved to account quintonh@gmail.com to make it visible to the public without any login.
Correct "push request" to "pull request".

thockin commented:
LGTM
